pkg/azuredisk/nodeserver.go

```diff
@@ -506,3 +461,33 @@ func (d *Driver) getBlockSizeBytes(devicePath string) (int64, error) {
 	}
 	return gotSizeBytes, nil
 }
+
+func (d *Driver) ensureBlockTargetFile(req *csi.NodePublishVolumeRequest, target string) (string, error) {
```

Comment:
there are two functionalities for this func:
 - find source
 - ensure mount point
I would suggest split it up since there are in relevant, and `ensure mount point` functionality is quite similar to `VolumeCapability_Mount`, only difference is partent dir vs current dir.

Reply:
split into 1) find source 2) ensure mount point and make file
and your function name is `ensureBlockTargetFile` which describe 2, 3, function #1 should not be included.

Reply:
done.